### DISCUSSION TITLE
Missing subdomain tab

### DISCUSSION THREAD

**Discussion by felixalguzman** [simple-login/app#764] [Self-hosting question] [answered]
I have a working simplelogin instance, but i don't see the subdomain tab shown in the [docs](https://simplelogin.io/docs/subdomain/new/). Is this normal ?

**Comment by nguyenkims** [comment, marked as answer]
@felixalguzman yes it's normal: the subdomain feature was developed after the latest release version. We'll make a new release soon which will include all new features.

**Reply by 0xspade** [reply]
Hi @nguyenkims,

When is the next update of the SimpleLogin?

**Reply by nguyenkims** [reply]
We don't have a fixed date for releasing a new version yet. It should happens soon though, we just need to make sure to thoroughly test the new version before making the release.

**Reply by 0xspade** [reply]
Take your time guys, BTW, submitted a vuln which is existing in the self-hosted as well as in the simplelogin, I submitted it to the hi@ email and forwarded it to the security@ email.

**Reply by nguyenkims** [reply]
@0xspade hey we haven't received this vulnerability email yet.

**Reply by 0xspade** [reply]
security@simplelogin.io is that right? Can you take a look at your Spam? I initially emailed it to hi@simplelogin.io and then forward it to security@

**Reply by nguyenkims** [reply]
The email does indeed end up in the Spam, thanks for reporting the issue!
